src/main/docs/guide/rest.adoc

```diff
@@ -0,0 +1,4 @@
+The MicroStream Storage isn’t a typical database server with administrative tooling, it is just a Java library which runs embedded in your application.
```

Comment:
lets rephrase this so that it sounds more appreciative of Microstream. 

Replace: 

`it is just a Java library`with `it is a pure java persistence engine`

Reply:
I copied that from their site 🤷